### PR TITLE
v6 release

### PR DESCRIPTION
**Pull request by Kikobeats**

### CODE REVIEW

File: benchmark/index.mjs [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import { readFile, writeFile } from 'fs/promises'
+import colorthief from 'colorthief'
```
Comment (pixelastic) [inline benchmark/index.mjs:2]: I think you're missing some of those dependencies (`colorthief`, `tinycolor2`, `lodash-es`) from your `devDependencies`.

File: package.json [hunk 2]
```diff
@@ -41,7 +41,9 @@
     "url"
   ],
   "dependencies": {
+    "@lokesh.dhakar/quantize": "~1.4.0",
     "debug-logfmt": "~1.2.3",
+    "ndarray": "~1.0.19",
     "node-addon-api": "~8.3.0",
     "node-gyp": "~11.0.0",
     "node-vibrant": "~3.2.1-alpha.1",
```
Comment (pixelastic) [inline package.json:49]: Do you still need this one?

File: benchmark/index.mjs [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+import Vibrant from 'node-vibrant'
+import tinycolor from 'tinycolor2'
+import { chain } from 'lodash-es'
+import { readdirSync } from 'fs'
+import mql from '@microlink/mql'
+import path from 'node:path'
+
+import splashy from '../src/index.js'
+
+const __dirname = import.meta.dirname
```
Comment (pixelastic) [inline benchmark/index.mjs:12]: This didn't work for me, I had to:

```
import { fileURLToPath } from 'node:url'
const __dirname = path.dirname(fileURLToPath(import.meta.url));
```

File: src/index.js [hunk 4]
```diff
@@ -1,34 +1,47 @@
...
+  const pixelArray = []
+
+  for (let i = 0, offset; i < pixelCount; i += quality) {
+    offset = i * 4
+    const r = pixels[offset]
+    const g = pixels[offset + 1]
+    const b = pixels[offset + 2]
+    const a = pixels[offset + 3]
+
+    if ((a === undefined || a >= 125) && !(r > 250 && g > 250 && b > 250)) {
```
Comment (pixelastic) [inline src/index.js:31]: I think this code excludes black and white pixels. Any reason why we're doing that?


image
image

File: package.json (hunk 2 above)
Comment (Kikobeats) [reply package.json:49]: yes, check https://github.com/microlinkhq/splashy/blob/c740654d367f587b34d37a316ca213b1346ba224/src/index.js#L7

File: src/index.js (hunk 4 above)
Comment (Kikobeats) [reply src/index.js:26]: This is interesting; it's in the original colorthief implementation
https://github.com/lokesh/color-thief/blob/master/src/color-thief-node.js#L17

**Pull request merged**